README.md

```diff
@@ -343,6 +344,15 @@ Found '/path/to/project/.nvmrc' with version <5.9>
 Now using node v5.9.1 (npm v3.7.3)
 ```
 
+### .node-version
+
+For a little compatability with other node version managers, nvm will also sniff for `.node-version` files. They're the same as `.rmvrc`, they just share a common name.
```

Comment:
`compatibility`

also there's no need to mention rvm; node and ruby don't have as much overlap as you might think :-)

Reply:
I meant nvm not rvm, sorry! :D

Reply:
in that case, they're not actually the same - nvmrc supports any "versionish" that nvm does; node-version only would support a literal major.minor.patch version.